application/src/main/java/org/togetherjava/tjbot/commands/moderation/scam/ScamBlocker.java

```diff
@@ -249,7 +249,7 @@ private void dmUser(Guild guild, long userId, JDA jda) {
                         This means you can no longer interact with anyone in the server until you have been unquarantined again.
 
                         If you think this was a mistake (for example, your account was hacked, but you got back control over it),
-                        please contact a moderator or admin of the server.
+                        you can get in touch with a moderator by using the **/modmail** command :thumbsup:
```

Comment:
gotta use slash-command mentions (`MessageUtils.slashCommandMention(...)`)

Reply:
Done

Reply:
Not a mention?

Reply:
fixed